api/all/src/main/java/io/opentelemetry/api/logs/LogRecordBuilder.java

```diff
@@ -101,6 +108,36 @@ default LogRecordBuilder setAllAttributes(Attributes attributes) {
   /** Sets an attribute. */
   <T> LogRecordBuilder setAttribute(AttributeKey<T> key, T value);
 
+  /** Sets a string attribute. */
+  default LogRecordBuilder setAttribute(String key, String value) {
+    return setAttribute(stringKey(key), value);
+  }
+
+  /** Sets a Long attribute. */
+  default LogRecordBuilder setAttribute(String key, Long value) {
+    return setAttribute(longKey(key), value);
+  }
+
+  /** Sets a Double attribute. */
+  default LogRecordBuilder setAttribute(String key, Double value) {
+    return setAttribute(doubleKey(key), value);
+  }
+
+  /** Sets a Boolean attribute. */
+  default LogRecordBuilder setAttribute(String key, Boolean value) {
+    return setAttribute(booleanKey(key), value);
+  }
+
+  /** Sets an Integer attribute. */
+  default LogRecordBuilder setAttribute(String key, Integer value) {
+    return setAttribute(key, value.longValue());
+  }
+
+  /** Sets a string array attribute. */
```

Comment:
Interestingly, neither `SpanBuilder` nor `Span` have explicit setters for the array values... 

Not sure why that would be.. I will say that if we want to do this, we should add support for all the primitive array types. I.e. `List<Long>`, `List<Double>`, `List<Boolean>`.

Reply:
I think I should drop it for this PR and we can see about revisiting/adding it later when the need arises. I was thinking that it might be useful, and the others less so....but definitely just overthinking it. I'll remove.